### PR TITLE
feat: inital move cli to mono-repo

### PR DESCRIPTION
**Pull request by danstarns**
Moving the cli: 

* https://github.com/GrapesJS/cli

Into the mono-repo.

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    "webpack-cli": "5.1.4",
+    "webpack-dev-server": "5.1.0",
+    "yargs": "^17.6.2"
+  },
+  "devDependencies": {
+    "@types/webpack-node-externals": "^3.0.0",
+    "copy-webpack-plugin": "^11.0.0",
+    "fork-ts-checker-webpack-plugin": "^8.0.0",
+    "webpack-node-externals": "^3.0.0"
+  }
```
Comment (danstarns) [inline packages/cli/package.json:56]: A lot of these packages are fairly out of date. I think we should keep them as they are, and before upgrading them add more comprehensive tests to the CLI so we update each with confidence.

File: packages/core/package.json [hunk 2]
```diff
@@ -67,13 +67,13 @@
   "scripts": {
     "build": "npm run build-all",
     "build-all": "run-s build:*",
-    "build:js": "grapesjs-cli build --patch=false --targets=\"> 1%, ie 11, safari 8, not dead\" --statsOutput=\"stats.json\" --localePath=\"src/i18n/locale\"",
-    "build:mjs": "BUILD_MODULE=true grapesjs-cli build --dts='skip' --patch=false --targets=\"> 1%, ie 11, safari 8, not dead\"",
+    "build:js": "node node_modules/grapesjs-cli/dist/cli.js build --patch=false --targets=\"> 1%, ie 11, safari 8, not dead\" --statsOutput=\"stats.json\" --localePath=\"src/i18n/locale\"",
```
Comment (danstarns) [inline packages/core/package.json:70]: Note now we: 

```
node_modules/grapesjs-cli/dist/cli.js
```

Point to the workspace version that's symbolically linked in the node modules dir of the core package.

File: packages/core/package.json [hunk 3]
```diff
@@ -41,7 +41,7 @@
   },
   "devDependencies": {
     "@types/markdown-it": "14.1.2",
-    "grapesjs-cli": "4.1.3",
+    "grapesjs-cli": "workspace:^",
```
Comment (danstarns) [inline packages/core/package.json:44]: Makes a symbolic link

**Comment by danstarns** [comment]
Now you can run: 

```
pnpm start
```

In the root. It will run the `packages/cli` against the `packages/core/_|index.html` file.

**Review by artf — APPROVED** [review]

**Pull request merged**